app/jobs/map_github_repositories_to_owners.py

```diff
@@ -86,18 +89,17 @@ def main(
     configure_logging(app_config.logging_level)
     logger.info("Running...")
 
-    database_service = DatabaseService()
     github_service = GithubService(app_config.github.token)
     repositories = github_service.get_all_repositories()
 
     for owner in owners:
+        logger.info(f"Mapping Repositories for Owner [ {owner} ]")
```

Comment:
Is it okay to print this out to the terminal?

Reply:
Should we protect it behind debug?

Reply:
Or is owner the business unit?

Reply:
The owner is a business unit - so nothing sensitive here 😁